packages/gafl-webapp-service/src/pages/licence-details/licence-length/update-transaction.js

```diff
@@ -12,12 +12,20 @@ import { licenceToStart } from '../licence-to-start/update-transaction.js'
  * @param permission
  */
 const checkContactDetails = permission => {
-  if (isPhysical(permission) && permission?.licensee?.preferredMethodOfConfirmation === mappings.HOW_CONTACTED.none) {
+  const preferredMethodOfConfirmation = permission?.licensee?.preferredMethodOfConfirmation
+  const preferredMethodOfReminder = permission?.licensee?.preferredMethodOfReminder
+  if (
+    isPhysical(permission) &&
+    (preferredMethodOfConfirmation === mappings.HOW_CONTACTED.none || preferredMethodOfReminder === mappings.HOW_CONTACTED.none)
```

Comment:
This, and the one underneath about postal contact, might be better extracted to well-named variables to make the intent a bit clearer

Reply:
So e.g.
```
const preferredMethodOfConfirmation = permission?.licensee?.preferredMethodOfConfirmation
const preferredMethodOfReminder = permission?.licensee?.preferredMethodOfReminder
const physicalContactNone = isPhysical(permission) &&
    (preferredMethodOfConfirmation === mappings.HOW_CONTACTED.none || preferredMethodOfReminder === mappings.HOW_CONTACTED.none)
if (physicalContactNone) {
...
}
    
```